executor_manager/routers/wegent_e2b_proxy.py

```
@@ -0,0 +1,392 @@
+            detail={"code": "timeout", "message": f"Request timeout: {str(e)}"},
+        )
+    except Exception as e:
+        logger.error(f"[SandboxProxy] Proxy error for sandbox {sandbox_id}: {e}")
+        raise HTTPException(
+            status_code=500,
+            detail={"code": "proxy_error", "message": str(e)},
+        )
+
+
+async def _proxy_subagent_execute(
+    base_url: str, sandbox, body: bytes, _request: Request
```

Ruff (0.14.10)

Unused function argument: `base_url`

(ARG001)